src/speed.c

```diff
@@ -36,15 +31,34 @@ struct turn_parameters {
 	int sign;
 };
 
+/**
+ * @brief Calculate the maximum search linear speed.
+ *
+ * This speed is calculated so that the search speed in long straight lines
+ * is always constant.
+ *
+ * @param[in] force Maximum force to apply while searching.
```

Comment:
- include @return parameter
- why the name of the function starts with _?
- could you explain the formula?

Reply:
- Why do we need to calculate the search linear speed but not the run linear speed?
- Do not the force parameter only apply for run? (or taking into account user configuration)

Reply:
- About the "@return": ecc88aa820b73783c039b23bd67e0a92a9f92119
- About the formula: #388
- `_` could be an indication of static functions (not consistent with what we have I know) :sweat_smile: